### PR TITLE
[FIX & FEATURE] fix bug with build & add id's on questions

### PR DESCRIPTION
**Pull request by fejes713**
## Description

Accidentally didn't test build script on #126 and merged it even if the build script didn't work. Reverted all changes back so everything works now.

As proposed in #114 I added id's to each question so people can now share questions simply by sharing `link/#question-name`

**Resolves** #114 

## What does your PR belong to?

* [ ] Questions / Answers
* [x] Website
* [ ] General / Things regarding the repository (like CI Integration)

## Types of changes

* [x] Bug fix (non-breaking change which fixes an issue)
* [x] Enhancement (non-breaking improvement of a question)
* [x] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

* [x] My code follows the code style of this project.
* [x] My change requires a change to the documentation.
* [x] I have updated the documentation accordingly.
* [x] I have checked that the changes are working properly
* [x] I have checked that there isn't any PR doing the same
* [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: website/js/components/Question.js [hunk 1]
```diff
@@ -19,6 +19,7 @@ export default ({
         tags.includes(state.filter.category.toLowerCase())
     })}
     key={name}
+    id={name.substring(0, name.length - 3)}
```
Comment (atomiks) [inline website/js/components/Question.js]: I prefer `slice()` but it's no big deal

Comment (fejes713) [reply website/js/components/Question.js]: Could you explain this one? I kinda feel lost using `.slice()` to cut last 3 characters. 😢

**Comment by fejes713** [comment]
Changes made! @atomiks I'll let you merge this one 🚀

**Comment by fejes713** [comment]
@atomiks the live version of the http://30secondsofinterviews.org doesn't work. Could you take a look at this, please?

**Comment by atomiks** [comment]
The cron hasn't run yet

**Comment by fejes713** [comment]
@flxwu how do we manually run cron job on Travis? What was the command?

**Comment by atomiks** [comment]
`--force-build` in the commit message

**Pull request merged**